app.js

```diff
@@ -8,6 +11,17 @@ fetch(urlApi)
 .then((data) => {
   console.log('Success:', data);
   recipeUrlElement.setAttribute("href", data.hits[0].recipe.url);
+  labelElement.innerHTML = data.hits[0].recipe.label;
+  imageElement.src = data.hits[0].recipe.image;
+  let ingredientList = data.hits[0].recipe.ingredientLines
+  // Since different cakes will have different amount of ingredients 
+  // then this codes create list items according to the number of ingredients
+  ingredientList.forEach(elem => {
+    let item = document.createElement("li");
```

Comment:
I would use const keyword. So actually wenn you a not going to reassign your variable, it's better to declare it as const.